src/layout/PageBody/Contain/Card/Card/Card.jsx

```diff
@@ -0,0 +1,9 @@
+import "./Card.css";
+
+export const Card = ({ children, onMouseOver, onMouseLeave }) => {
+  return (
+    <div className="Card" onMouseOver={onMouseOver} onMouseLeave={onMouseLeave}>
+      {children}
+    </div>
+  );
+};
```

Comment:
```suggestion
export const Card = (props) => {
  return (
    <div
      {...props}
      className="Card" 
      />
  );
};
```

DOM이 기본적으로 가진 prop을 사용하는 경우라면 확장성 측면에서 스프레드해주는게 더 사용성이 좋습니다.

스프레드에 대해 저도 기본적으로 부정적인 입장이지만 이는 type을 제공받을 수 있으면 충분히 대응가능한 영역이기 때문에 위 방식을 권장드립니다.

타입을 어떻게 활용할 수 있는지와 스프레드 사용 기준에 대한 내용은 멘토링 시간에 보완설명하도록 하겠습니다.

Reply:
넵 알겠습니다.